docs/identities/sign-in/actions.mdx

```diff
@@ -125,3 +130,67 @@ created account to get access to your services, but instead can access all the f
 
 1. Go to Ory Console → [**Base Settings**](https://console.ory.sh/projects/current/auth).
 2. Toggle **Sign In After Registration**.
+
+## Show verification after successful registration
+
+If you want to show the verification screen after registration, follow these steps:
+
+```mdx-code-block
+<Tabs>
+  <TabItem value="Server rendered browser client">
+```
+
+For server rendered applications, such as an Express.js app or when using the Ory Account Experience, Ory Identities can be
+configured to redirect to the verification screen automatically after registration:
+
+```shell
+ory patch identity-config {project_id} \
+  --add '/selfservice/flows/registration/after/hooks=[{"hook": "show_verification_ui"}]'
+```
+
+:::note
+
+If your identity schema defines multiple verifiable addresses, Ory Identities redirects to the verification flow of only one of
+the addresses.
+
+:::
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="SPA & Native clients">
+```
+
+For SPA & native clients, the response from the registration endpoint contains a `continue_with` field. This field lists possible
+actions, the user might need to or can take next. For example, an object containing the ID of the verification flow created as
+part of the registration flow:
+
+```json
+{
+  "continue_with": [
+    {
+      "action": "verification_ui",
+      "flow": {
+        "id": "d859f6af-1dfe-453e-9320-d572e10edeea",
+        "verifiable_address": "example@ory.sh"
+      }
+    }
+    // Other items
+  ]
+}
+```
+
+You can use this ID to fetch the verification flow information, using the
+[`getVerificationFlow`](../../reference/api#tag/frontend/operation/getVerificationFlow) API method.
+
+:::note
+
+The response to the registration endpoint always contains the `continue_with` field with the `verification_ui` action if
+verification is enabled and the identity schema defines at least one verifiable address. Due to this, the `show_verification_ui`
+hook does nothing for API & SPA clients.
```

Comment:
```suggestion
The response from the registration endpoint always contains the `continue_with` field with the `verification_ui` action, if
verification is enabled and the identity schema defines at least one verifiable address. Because of this, the `show_verification_ui`
hook has no effect for API and SPA clients.
```

Reply:
Done.